pages/start_page.py

```diff
@@ -11,59 +12,67 @@ def __init__(self, driver):
-        sleep(1)
 
     def verify_sign_up_email_error(self, email_error, email_log, email_element):
         assert self.get_element_text(email_element) == email_error, f"Actual message: {self.get_element_text(email_element)}"
         self.log.info(email_log)
-        sleep(1)
 
     def sign_up_password(self, pwd):
         self.fill_field(xpath=self.constants.SIGN_UP_PASSWORD_FIELD_XPATH, value=pwd)
         self.log.info(f"'{pwd}' entered to password field")
-        sleep(1)
 
     def verify_sign_up_password_error(self, pwd_error, pwd_log, pwd_element):
         assert self.get_element_text(pwd_element) == pwd_error, f"Actual message: {self.get_element_text(pwd_element)}"
         self.log.info(pwd_log)
-        sleep(1)
 
-    def sign_up(self, username, email, pwd):
+    @wait_until_ok(period=0.5)
+    def register(self, username, email, pwd):
         # Fill username, email, password
         self.fill_field(xpath=self.constants.SIGN_UP_USERNAME_FIELD_XPATH, value=username)
         self.log.info(f"'{username}' entered to username field")
         self.fill_field(xpath=self.constants.SIGN_UP_EMAIL_FIELD_XPATH, value=email)
         self.log.info(f"'{email}@gmail.com' entered to email field")
         self.fill_field(xpath=self.constants.SIGN_UP_PASSWORD_FIELD_XPATH, value=pwd)
         self.log.info(f"'{pwd}' entered to password field")
-        sleep(1)
         # Click button
-        self.click(xpath=self.constants.SIGN_UP_BUTTON_XPATH)
-        self.log.info(f"button 'Sign up for OurApp' clicked")
         sleep(1)
+        self.click_sign_up_and_verify()
+        self.log.info(f"button 'Sign up for OurApp' clicked")
+        from pages.header import Header
+        return Header(self.driver)
 
-    def verify_profile_button(self):
-        """Verify success Sign Up using 'My profile' button"""
-        assert self.get_element(self.constants.MY_PROFILE_BUTTON_XPATH).is_enabled, f"Registration failed"
-        sleep(1)
+    @wait_until_ok(period=0.5)
+    def click_sign_up_and_verify(self):
+        # Click Sign Up button and verify
+        self.click(xpath=self.constants.SIGN_UP_BUTTON_XPATH)
+        assert not self.is_exists(xpath=self.constants.SIGN_UP_BUTTON_XPATH)
 
-    # TODO: next page
-    def sign_out_click(self):
-        self.click(xpath=self.constants.SIGN_OUT_BUTTON_XPATH)
-        self.log.info("button 'Sign Out' clicked")
-        sleep(1)
+    def sign_in(self, username, password):
+        # Fill login
+        self.fill_field(xpath=self.constants.SIGN_IN_USERNAME_XPATH, value=username)
+        self.log.info(f"'{username}' entered to username field")
+        self.fill_field(xpath=self.constants.SIGN_IN_PASSWORD_XPATH, value=password)
+        self.log.info(f"'{password}' entered to password field")
+        self.click(xpath=self.constants.SIGN_IN_BUTTON_XPATH)
+        self.log.info(f"button 'Sign In' clicked")
+        from pages.header import Header
+        return Header(self.driver)
```

Comment:
It should returns new page, not element. 
In my project I'm calling it HelloPage